lambdas/common/ecs_utils.py

```diff
@@ -88,3 +88,50 @@ def clone_task_definition(client, task_definition):
     )
 
     return new_task['taskDefinition']['taskDefinitionArn']
+
+
+def get_cluster_arns(client):
+    """
+    Extract the list of cluster ARNs in this account.
+
+    Returns a list of cluster ARNs.
+    """
+    response = client.list_clusters()
+
+    cluster_arns = []
+
+    [cluster_arns.append(cluster_arn) for cluster_arn in response['clusterArns']]
```

Comment:
Close, but better would be to assign directly rather than relying on side effects, i.e.:

```python
cluster_arns = [cluster_arn for cluster_arn in response['clusterArns']]
return cluster_arns
```

although at that point you could skip the local variable entirely.